pypush/apns/protocol.py

```diff
@@ -201,18 +197,14 @@ def __post_init__(self):
         else:
             assert self._token_topic_1 and self._token_topic_2
             if len(self._token_topic_1) == 20:  # SHA1 hash, topic
-                self.topic = (
-                    KNOWN_TOPICS_LOOKUP[self._token_topic_1]
-                    if self._token_topic_1 in KNOWN_TOPICS_LOOKUP
-                    else self._token_topic_1
+                self.topic = KNOWN_TOPICS_LOOKUP.get(
```

Comment:
This possibly duality of `topic` depending on if we know it or not could come back to bite us

Reply:
FIne for now, will deal with it when I hit any issues